.github/workflows/publish.yml

```diff
@@ -11,17 +11,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v5
-      - name: Set up Python 3.x
-        uses: actions/setup-python@v6
+      
+      - name: Install uv
+        uses: astral-sh/setup-uv@v7
         with:
-          python-version: "3.x"
-      - name: Install release dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install setuptools wheel twine
+          version: "0.9.*" 
+          enable-cache: true
+    
       - name: Build and publish package
         env:
-          TWINE_USERNAME: ${{ secrets.PYPI_STACUTILS_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_STACUTILS_PASSWORD }}
+          UV_PUBLISH_USERNAME: ${{ secrets.PYPI_STACUTILS_USERNAME }}
+          UV_PUBLISH_PASSWORD: ${{ secrets.PYPI_STACUTILS_PASSWORD }}
```

Comment:
Not needed for this PR, but could be good to set up https://docs.pypi.org/trusted-publishers/ at some point